bouncer/shared/provide_liquidity.ts

```diff
@@ -15,13 +15,18 @@ import {
 } from '../shared/utils';
 import { send } from '../shared/send';
 
-export async function provideLiquidity(ccy: Asset, amount: number, waitForFinalization = false) {
+export async function provideLiquidity(
+  ccy: Asset,
+  amount: number,
+  waitForFinalization = false,
+  lpKey = '',
```

Comment:
Nit: you can use `lpKey?: string`, which will automatically have the default value of `undefined`, and then later you can do:
```ts
  const lpUri = lpKey ?? process.env.LP_URI || '//LP_1';
```
It is a more ideomatic way of using "optional" values in JS.